frappe/tests/test_db.py

```diff
@@ -24,8 +24,8 @@ def test_get_value(self):
 		self.assertNotEqual(frappe.db.get_value("User", {"name": ["!=", "Guest"]}), "Guest")
 		self.assertEqual(frappe.db.get_value("User", {"name": ["<", "Adn"]}), "Administrator")
 		self.assertEqual(frappe.db.get_value("User", {"name": ["<=", "Administrator"]}), "Administrator")
-		self.assertEqual(frappe.db.get_value("User", {}, ["Max(name)"]), frappe.db.sql("SELECT Max(name) FROM tabUser")[0][0])
-		self.assertEqual(frappe.db.get_value("User", {}, "Min(name)"), frappe.db.sql("SELECT Min(name) FROM tabUser")[0][0])
+		self.assertEqual(frappe.db.get_value("User", {}, ["Max(name)"], order_by=True), frappe.db.sql("SELECT Max(name) FROM tabUser")[0][0])
+		self.assertEqual(frappe.db.get_value("User", {}, "Min(name)", order_by=True), frappe.db.sql("SELECT Min(name) FROM tabUser")[0][0])
```

Comment:
```suggestion
		self.assertEqual(frappe.db.get_value("User", {}, ["Max(name)"], order_by=None), frappe.db.sql("SELECT Max(name) FROM tabUser")[0][0])
		self.assertEqual(frappe.db.get_value("User", {}, "Min(name)", order_by=None), frappe.db.sql("SELECT Min(name) FROM tabUser")[0][0])
```

Didn't understand the `True`? Did you mean `None`?

Reply:
yes I did 😢

Reply:
fixed in 62fc126